### PR TITLE
Modelling fix

### PR DESCRIPTION
**Pull request by tmsean**
Fix syntax
Rename column

### CODE REVIEW

File: app/models/word.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Word < ActiveRecord::Base
+	has_many :results
```
Comment (trantube94) [inline app/models/word.rb]: ???

File: db/migrate/20160713092146_change_column_name.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class ChangeColumnName < ActiveRecord::Migration
+  def change
+  	rename_column :users, :user_name, :name
```
Comment (trantube94) [inline db/migrate/20160713092146_change_column_name.rb]: ?

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  has_many :active_relationships, class_name:  "Relationship",
```
Comment (trantube94) [inline app/models/user.rb]: indent

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    foreign_key: "follower_id", dependent: :destroy
+  has_many :passive_relationships, class_name:  "Relationship",
```
Comment (trantube94) [inline app/models/user.rb]: indent

**Comment by trantube94** [comment]
gop 3 commit lai

**Comment by tmsean** [comment]
ok a hqua e có việc bận chưa check mail được để bh ngồi sửa lại :v

2016-07-13 22:38 GMT+07:00 Trần Thanh Tùng <notifications@github.com>:

> gop 3 commit lai
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sachmem/els4/pull/3#issuecomment-232394950>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AQ4VIpTlZZSTsddFP8v__i3QZTx_3DxOks5qVQZsgaJpZM4JLNQ6>
> .
>



-- 
Tô Minh Sơn
Mã SV: USTHBI5-119
SĐT 0163 228 3836
Trường ĐH Khoa học và Công nghệ Hà Nội (USTH)

**Pull request closed without merging**